templates/municipal_chief/home.html.twig

```diff
@@ -86,7 +86,21 @@
                     <a href="https://chezvous.en-marche.fr/" target="_blank" class="btn btn btn--ghosting--blue">Consulter</a>
                 </footer>
             </div>
-            <div></div>
+            {% if app.user.municipalChiefManagedArea.hasJecouteAccess() %}
+                <div class="manager-overview__case">
+                    <span class="manager-overview__case--section">Communication & Mobilisation</span>
+                    <h1 class="manager-overview__case--title">J'écoute</h1>
+                    <p class="manager-overview__case--baseline">Application disponible sur
+                        <a class="text--blue--dark link--no-decor" target="_blank" href="https://play.google.com/store/apps/details?id=fr.en_marche.jecoute&hl=it">Google Play</a>
```

Comment:
`...play.google.com/....&hl=it` 
`hl=it` it's for the Italian version of the page 🇮🇹 🍝

Reply:
hahaha correct